app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list) -> list:
+    person_list = []
+    for human in people_list:
+        person = Person(human["name"], human["age"])
+        person_list.append(person)
+
+    for human in people_list:
+        if human.get("wife"):
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        if human.get("husband"):
+            husband_link = Person.people[human["husband"]]
+            Person.people[human["name"]].husband = husband_link
```

Comment:
it is better to make separate varaible, for similar part of conditions f.e. current_person = Person.people[human["name"]]

Reply:
thanks, it'll be clearer